frontend/src/components/uam/helpers/get-rows.helper.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  UsersGetAllItemResponseDto,
+  UsersGetAllResponseDto,
+} from 'guruhub-shared';
+
+const getRows = (
+  users: UsersGetAllResponseDto,
+): readonly UsersGetAllItemResponseDto[] => {
+  return users.map((user) => ({
+    id: user.id,
+    email: user.email,
```

Comment:
```suggestion
    [UserTableAccesson.ID]: user.id,
    email: user.email,
```
create an enum for all keys